src/content/en/2020/compression.md

```diff
@@ -4,20 +4,271 @@ part_number: IV
 chapter_number: 19
 title: Compression
 description: Compression chapter of the 2020 Web Almanac covering HTTP compression, algorithms, content types, 1st party and 3rd party compression and opportunities.
-authors: [mo271]
-reviewers: [jyrkialakuijala, sboukortt, veluca93, paulcalvano]
+authors: [mo271, veluca93, sboukortt, jyrkialakuijala]
+reviewers: [paulcalvano]
 analysts: [AbbyTsai]
 translators: []
-#mo271_bio: TODO
+jyrkialakuijala_bio: Jyrki Alakuijala works as a software engineer at Google Switzerland.
+sboukortt_bio: Sami Boukortt works as a software engineer at Google Switzerland.
+mo271_bio: Moritz Firsching works as a software engineer at Google Switzerland.
+veluca93_bio: Luca Versari works as a software engineer at Google Switzerland.
 discuss: 2055
 results: https://docs.google.com/spreadsheets/d/1NKbP4AqMkgCNCsVD3yLhO2d0aqIsgZ7AGLEtUDHl9yY/
 queries: 19_Compression
-#featured_quote: TODO
-#featured_stat_1: TODO
-#featured_stat_label_1: TODO
-#featured_stat_2: TODO
-#featured_stat_label_2: TODO
-#featured_stat_3: TODO
-#featured_stat_label_3: TODO
-unedited: true
+featured_quote: Using HTTP compression makes a website load faster and therefore guarantees a better user experience.
+#featured_stat_1: 23%
+#featured_stat_label_1: Compressed requests which use brotli
+#featured_stat_2: 77%
+#featured_stat_label_2: Compressed requests which use gzip
+#featured_stat_3: 74%
+#featured_stat_label_3: Websites that pass the Lighthouse audit with maximum score on text compression
```

Comment:
Uncommenting the featured stats, and re-adding the `unedited` label, which is our final check for copyediting (after peer-review)
```suggestion
featured_quote: Using HTTP compression makes a website load faster and therefore guarantees a better user experience.
featured_stat_1: 23%
featured_stat_label_1: Compressed requests which use brotli
featured_stat_2: 77%
featured_stat_label_2: Compressed requests which use gzip
featured_stat_3: 74%
featured_stat_label_3: Websites that pass the Lighthouse audit with maximum score on text compression
unedited: true
```